packages/preview/typst-railroad/0.1.0/lib.typ

```diff
@@ -0,0 +1,30 @@
+#let rr-wasm = plugin("./typst_railroad.wasm")
+
+#let _as_bytes(input) = {
+    let t = type(input)
+    if t == "bytes" {
+        return input
+    }
+    if t == "string"{
+        return bytes(input)
+    }
+    if input.has("text") {
+        return _as_bytes(input.text)
+    }
+    assert(true, message:  "The input is not string, bytes, or a content with text")
+}
+
+
+
+#let render(
+    railroad,
+    css: rr-wasm.default_css(),
+    ..args
+) = {
+    image.decode(rr-wasm.railroad(_as_bytes(railroad), _as_bytes(css)), format: "svg")
+}
+
+
+#let default_css() = {
```

Comment:
kebab-case (i.e `default-css`) is a more common convention in the Typst ecosystem (but snake_case is allowed, if you prefer to keep it).

Reply:
👍